docs_src/microservices/application/V2Migration.md

```diff
@@ -289,10 +289,10 @@ Custom profiles used with App Service Configurable are configuration files. Thes
 5. `BatchByCount`, `BatchByTime` and `BatchByTimeAndCount` have been collapsed into `Batch` with additional parameters. See the [Batch](../AppServiceConfigurable/#batch) section for more details.
 6. `SetOutputData` has been renamed to `SetResponseData`. See the [SetResponseData](../AppServiceConfigurable/#setresponsedata) section for more details.
 7. `PushToCore` parameters have changed. See the [PushToCore](../AppServiceConfigurable/#pushtocore) section for more details.
-9. `HTTPPost`, `HTTPPostJSON`, `HTTPPostXML`, `HTTPPut`, `HTTPPutJSON` and `HTTPPutXML` have been collapsed into `HTTPExport` with additional parameters. See the [HTTPExport](../AppServiceConfigurable/#httpexport) section for more details.
-10. `MQTTSecretSend` has been renamed to `MQTTExport` with additional parameters. See the [MQTTExport](../AppServiceConfigurable/#mqttexport) section for more details.
-11. `MarkAsPushed` has been removed.
-11. `MQTTSend` has been removed.
+8. `HTTPPost`, `HTTPPostJSON`, `HTTPPostXML`, `HTTPPut`, `HTTPPutJSON` and `HTTPPutXML` have been collapsed into `HTTPExport` with additional parameters. See the [HTTPExport](../AppServiceConfigurable/#httpexport) section for more details.
+9. `MQTTSecretSend` has been renamed to `MQTTExport` with additional parameters. See the [MQTTExport](../AppServiceConfigurable/#mqttexport) section for more details.
+10. `MarkAsPushed` has been removed. The mark as push capability has been removed from Core Data, which this depended on.
+11. `MQTTSend` has been removed. This has been replaced by `MQTTExport`
```

Comment:
Might want to make it a link `[MQTTExport](../AppServiceConfigurable/#mqttexport)`

Reply:
Added